docs/source/concepts/bento.rst

```diff
@@ -683,6 +683,16 @@ Here's a basic Docker options configuration.
             - FOO=value1
             - BAR=value2
 
+.. note::
+
+   BentoML leverage `BuildKit <https://github.com/moby/buildkit>`_, a cache-efficient builder toolkit,
+   to containerize a Bento 🍱.
+
+   BuildKit comes with `Docker 18.09 <https://docs.docker.com/develop/develop-images/build_enhancements/>`_. This means
+   if you are using Docker via Docker Desktop, BuildKit will be available by default.
+
+   However, if you are using a standalone version of Docker, you can install
+   BuildKit by following the instructions `here <https://github.com/docker/buildx#installing>`_.
```

Comment:
This is great. I think it might be beneficial to raise a specific error for missing buildkit programmatically as well. What you suggested with `docker buildx ls` or any other sanity check sounded good to me.